tools/cldr-apps/js/src/esm/cldrAnnounce.mjs

```diff
@@ -68,36 +70,64 @@ async function refresh(viewCallbackSetData) {
   if (schedule.tooSoon()) {
     return;
   }
-  const url = cldrAjax.makeApiUrl("announce", null);
+  let p = null;
+  if (alreadyGotId) {
+    p = new URLSearchParams();
+    p.append("alreadyGotId", alreadyGotId);
+  }
+  const url = cldrAjax.makeApiUrl("announce", p);
   schedule.setRequestTime();
   return await cldrAjax
     .doFetch(url)
     .then(cldrAjax.handleFetchErrors)
     .then((r) => r.json())
     .then(setPosts)
-    .catch((e) => console.error(e));
+    .catch(handleErrorOrNotModified);
+}
+
+function handleErrorOrNotModified(e) {
+  if (e.message === "Not Modified") {
```

Comment:
```suggestion
  if (e.status === 304) {
```

should be able to check the status code directly - the message can be variable.

Reply:
That would require changes to cldrAjax -- currently cldrAjax.handleFetchErrors is as follows:

```
function handleFetchErrors(response) {
  if (!response.ok) {
    throw new Error(response.statusText);
  }
  return response;
}
```
The Error only contains response.statusText -- I'm not sure how to set e.status in a standard way